app/dao/services_dao.py

```diff
@@ -72,6 +75,93 @@ def dao_count_live_services():
     ).count()
 
 
+def dao_fetch_live_services_data():
+    year_start_date, year_end_date = get_current_financial_year()
+    this_year_ft_billing = FactBilling.query.filter(
+        FactBilling.bst_date >= year_start_date,
+        FactBilling.bst_date <= year_end_date,
+    ).subquery()
+    data = db.session.query(
+        Service.id,
+        Organisation.name.label("organisation_name"),
+        Service.name.label("service_name"),
+        Service.consent_to_research,
+        Service.go_live_user_id,
+        Service.count_as_live,
+        User.name.label('user_name'),
+        User.email_address,
+        User.mobile_number,
+        Service.go_live_at.label("live_date"),
+        Service.volume_sms,
+        Service.volume_email,
+        Service.volume_letter,
+        case([
+            (this_year_ft_billing.c.notification_type == 'email', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("email_totals"),
+        case([
+            (this_year_ft_billing.c.notification_type == 'sms', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("sms_totals"),
+        case([
+            (this_year_ft_billing.c.notification_type == 'letter', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("letter_totals"),
+    ).outerjoin(
+        Service.organisation
+    ).outerjoin(
+        this_year_ft_billing, Service.id == this_year_ft_billing.c.service_id
+    ).outerjoin(
+        User, Service.go_live_user_id == User.id
+    ).filter(
+        Service.count_as_live == True  # noqa
+    ).group_by(
+        Service.id,
+        Organisation.name,
+        Service.name,
+        Service.consent_to_research,
+        Service.count_as_live,
+        Service.go_live_user_id,
+        User.name,
+        User.email_address,
+        User.mobile_number,
+        Service.go_live_at,
+        Service.volume_sms,
+        Service.volume_email,
+        Service.volume_letter,
+        this_year_ft_billing.c.notification_type
+    ).all()
```

Comment:
Do we need to order the results? Maybe by go_live_at?

Reply:
That's a good idea 👍